plenum/test/node_catchup/test_large_catchup.py

```diff
@@ -39,14 +40,18 @@ def reset():
     request.addfinalizer(reset)
     return tconf
 
+CHK_FREQ = 5
```

Comment:
I think checkpoints are not related to this test. The test is about catching up big ledgers. 
We have a limit for max message size, so if the ledger is huge, then we need to split the response into multiple parts. And this is the test that checks it.